src/Maestro/maestro-angular/src/app/page/build-graph-table/popIncoherencies.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input } from '@angular/core';
+import { Build, BuildIncoherence } from 'src/maestro-client/models';
+
+@Component({
+  selector: 'popIncoherencies',
+  templateUrl: './popIncoherencies.component.html'
+})
+
```

Comment:
NIT: There shouldn't be an empty line here. The `@Component()` thingy is a decorator, the equivalent of a C# attribute. It should be attached to the class it is applied to.

Reply:
Thanks. Fixed.